htdocs_symfony/src/Controller/App/IndexController.php

```diff
@@ -1,18 +1,24 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Oc\Controller\App;
 
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
+/**
+ *
+ */
```

Comment:
Wofür ist der Kommentar den gut? :-)

Reply:
Das Einfügen des Kommentars verschlankt die Liste von Problemen, die phpStorm in einer Datei anzeigt. Das verschafft mir die Übersichtlichkeit, mich auf die eigentlichen Probleme zu konzentrieren. Ich kann ja den Kommentar noch mit Leben füllen?

Reply:
Was zeigt dir den da PHPStorm genau an? Das ist ja nicht die Lösung einen Hinweis damit zu lösen, dass man einen leeren Kommentar erstellt.

Reply:
Die Meldung vom phpStorm lautet "Missing PHPDoc comment", als Warnung, kein Fehler.

Reply:
Ok, ich würde vorschlagen, dass ich nach meinem Urlaub den PHP-CS Fixer und phpstan in die CI Pipeline integrieren. Dann werden die Sachen automatisch geprüft und Vorschläge zum anpassen gemacht. Dann haben wir da keinen manuellen Aufwand mehr.